### PR TITLE
TLS error fix (if deploy to azure)

### PR DESCRIPTION
**Pull request by TiagoBrenck**
Solving issue: https://github.com/Azure-Samples/active-directory-dotnet-daemon/issues/39
Added to ServicePointManager.SecurityProtocol = SecurityProtocolTypeTls12
Without this, the app doesnt work when deployed to azure

### CODE REVIEW

**Comment by TiagoBrenck** [comment]
@jmprieur Changing the .NET Framework version didn't fix the problem. I changed both projects to run on 4.7.2 and the error occurred on the deployed version.

**Comment by TiagoBrenck** [comment]
@jmprieur I have updated the projects to run on .NET 4.7.2 and it worked. Pushed new changes

File: TodoListDaemon/Program.cs [hunk 1]
```diff
@@ -139,7 +140,8 @@ static async Task PostTodo()
             Console.WriteLine("Posting to To Do list at {0}", timeNow);
             string todoText = "Task at time: " + timeNow;
             HttpContent content = new FormUrlEncodedContent(new[] { new KeyValuePair<string, string>("Title", todoText) });
-            HttpResponseMessage response = await httpClient.PostAsync(todoListBaseAddress + "/api/todolist", content);
+
```
Comment (jmprieur) [inline TodoListDaemon/Program.cs]: tabs => spaces, please

File: TodoListDaemon/Program.cs [hunk 2]
```diff
@@ -140,8 +140,7 @@ static async Task PostTodo()
             Console.WriteLine("Posting to To Do list at {0}", timeNow);
             string todoText = "Task at time: " + timeNow;
             HttpContent content = new FormUrlEncodedContent(new[] { new KeyValuePair<string, string>("Title", todoText) });
-
-			HttpResponseMessage response = await httpClient.PostAsync(todoListBaseAddress + "/api/todolist", content);
+            HttpResponseMessage response = await httpClient.PostAsync(todoListBaseAddress + "/api/todolist", content);
```
Comment (jmprieur) [inline TodoListDaemon/Program.cs]: Thanks @TiagoBrenck

**Pull request merged**